PARAMETERS_MAPPING/parameters.csv

```diff
@@ -196,4 +196,17 @@ unique_id;cf_standard_name;cf_alias_name;long_name;imos_vocabulary_name;imos_voc
 201;;;hail_intensity_10min_sample;;;
 202;thickness_of_rainfall_amount;;rainfall_amount_since_9am;;;
 203;;;rainfall duration;;;
-204;rainfall_rate;;rainfall intensity_10min_sample;;;
\ No newline at end of file
+204;rainfall_rate;;rainfall intensity_10min_sample;;;
+850;number_concentration_of_biological_taxon_in_sea_water;;total abundance;Abundance of biota;;
+851;;;cell volume;Biovolume;;Mean size of cell
+852;;;length;;;Average total length
+853;mass_concentration_of_zooplankton_expressed_as_carbon_in_sea_water;;total zooplankton carbon;;;
+854;;;Shannon evenness;;;
+855;;;ratio;;;
+856;mass_concentration_of_diazotrophic_phytoplankton_expressed_as_carbon_in_sea_water;;total phytoplankton carbon;;;
+857;;;region;;;
```

Comment:
Claire had 'Region as defined for segment' in her document.

Reply:
yes, this would actually go in 'comments', but I am still not sure where to include them because some are not related to the parameter in general. Is there any other column that extra comments could be added to the paramater_mapping?

Reply:
Again, put it in long_name

Reply:
ok, for that one it will work fine. But there are a few pre-existing parameters, that need a specific comment for them. For example: (cf_standard names and comments are in bold)

data_product_name | data_column_name | **cf_standard_name** | long_name | imos_vocabulary_name | **comment** | unit_name | unit_short_name
-- | -- | -- | -- | -- | -- | -- | --
Derived Indices | ChlorophyllMonthlyClimatology_mg_m3 | **mass_concentration_of_chlorophyll_in_sea_water** |  |  | **Chlorophyll climatology for segment from MODIS 8 day 4 km monthly** | Milligrams per cubic metre | mg m-3
Derived Indices | ChlorophyllSatellite_mg_m3 | **mass_concentration_of_chlorophyll_in_sea_water** |   |   | **The best available Chlorophyll for segment from MODIS 4 km, 8 day, MODIS 4 km, monthly** | Milligrams per cubic metre | mg m-3
Derived Indices | SST_C | **sea_surface_temperature** |   | Temperature of the water body | **Sea surface temperature for segment from GHRSST for post 2009 data, Reynolds weekly average from NOAA for pre 2009 data** | Degrees Celsius | degC
Derived Indices | WaterDepth_m | **sea_floor_depth_below_sea_surface** |   | Sea-floor depth below surface of the water body | **Bathymetry for segment from Geosciences Australia via SDODE** | Metres | m

Reply:
You'll have to add "new" parameters for these ones then.

Reply:
I would just create new parameters, and put all those "comments" in the long_name column. Maybe it's not ideal, but it'll do.

Reply:
>  put all those "comments" in the long_name column
Why not using the _comment_ column specifically dedicated to cover these uses cases?

Reply:
I am confused, I will create new parameters for them, and then add the extra info in long-name or comments?

Reply:
I added it in the comments for now, we can change later if that is the case.

Reply:
Two more questions and then it is all done! 
1. Claire had as comments for variables like ammonium, nitrate, oxygen, phosphate, salinity, silicate, alkalinity and carbon the following: Average 'nitrate' from NRS bottle samples (as an example for nitrate, but all of them have the same). Is it necessary to include this info as comments/long_name as well? Because then I will need to add new variables for all of them too.
2. what are the units for variables like 'NoCopepodSpecies_sample' which are in Number of species / sample? does it need a unit, because there is no option for that in the units list

Reply:
> Why not using the comment column specifically dedicated to cover these uses cases? 

It just seemed a shame to leave the long name column blank when there's meaningful information you could put in there. Way over in the comments column, the user might not notice these details in the header... 

I don't really mind either way. I guess it's a somewhat arbitrary boundary between what details are part of the _name_ of a parameter and what is additional information (i.e. comments).

Reply:
> Claire had as comments for variables like ammonium, nitrate, oxygen, phosphate, salinity, silicate, alkalinity and carbon the following: Average 'nitrate' from NRS bottle samples (as an example for nitrate, but all of them have the same). Is it necessary to include this info as comments/long_name as well?

Yes, I would.

Reply:
> what are the units for variables like 'NoCopepodSpecies_sample' which are in Number of species / sample

As I said in my email, to me it's a dimensionless property, so it doesn't need untils

Reply:
ok, will just leave it without units